orca-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlConfiguration.kt

```diff
@@ -73,7 +73,8 @@ class SqlConfiguration {
     orcaSqlProperties: OrcaSqlProperties,
     interlink: Optional<Interlink>,
     executionRepositoryListeners: Collection<ExecutionRepositoryListener>,
-    compressionProperties: ExecutionCompressionProperties
+    compressionProperties: ExecutionCompressionProperties,
+    @Value("\${execution-repository.sql.pipeline-ref.enabled:false}") pipelineRefEnabled: Boolean
```

Comment:
I'm not a fan of `@Value` since they can be hard to find. Can you add a new config class (or a new member to an existing class) for this please?

Reply:
sure, I doubted on creating a config class because it is only a single boolean but I will add it to an existing config class

Reply:
@edgarulg I still see `@Value` here...  By the way, is the config knob here completely independent from compression?

Reply:
Yes, the pipelineRef config is independent to the compression feature. Let me remove the `@Value`